tfhe/src/high_level_api/integers/signed/tests.rs

```diff
@@ -648,11 +648,14 @@ fn test_safe_deserialize_conformant_fhe_int32() {
     let clear_a = random::<i32>();
     let a = FheInt32::encrypt(clear_a, &client_key);
     let mut serialized = vec![];
-    assert!(crate::safe_serialize(&a, &mut serialized, 1 << 20).is_ok());
+    assert!(SerializationConfig::new(1 << 20)
+        .serialize_into(&a, &mut serialized)
+        .is_ok());
```

Comment:
I think all `assert!(_.is_ok())` can be replaced by `unwrap`
Also applies to other files